### DISCUSSION TITLE
mac os13.3.1系统 m2芯片安装报错问题

### DISCUSSION THREAD

**Discussion by li330273334** [PaddlePaddle/PGL#557] [Q&A]
安装找不到对应版本的安装包，是否支持m2芯片安装
`pip install pgl`
`Looking in indexes: https://mirrors.aliyun.com/pypi/simple/`
`ERROR: Could not find a version that satisfies the requirement pgl (from versions: none)`
`ERROR: No matching distribution found for pgl`

**Comment by Yelrose** [comment]
可以先直接在PGL目录下，python setup.py install 来安装试一下。

**Reply by li330273334** [reply]
好的  谢谢  已经安装好了
